src/bin/opensmt.cc

```diff
@@ -238,4 +243,42 @@ void catcher( int sig )
   }
 }
 
+
+void printHelp( )
```

Comment:
Can you please update the help message so that it mentions all supported options and what they do?

Reply:
Fixed